apps/masterbots.ai/lib/hooks/use-profile.tsx

```
@@ -0,0 +1,77 @@
+
+export function ProfileProvider({ children }: ProfileProviderProps) {
+  const { data: session } = useSession()
+
+  const getuserInfo = async (slug: string): Promise<any> => {
+    if (!slug?.trim()) {
+      throw new Error('Slug is required')
+    }
+    try {
+      const userInfo = await getUserBySlug({
+        slug,
+        isSameUser: session?.user.slug == slug
```

Biome

[error] Use === instead of ==. == is only allowed when comparing against `null`

== is only allowed when comparing against null

Using == may be unsafe if you are relying on type coercion
Unsafe fix: Use ===


(lint/suspicious/noDoubleEquals)